docs/source/tutorial/dcgan.rst

```diff
@@ -0,0 +1,358 @@
+DCGAN: Generate the images with Deep Convolutional GAN
+******************************************************
+
+.. currentmodule:: chainer
+
+0. Introduction
+================
+
+In this tutorial, we generate images with **generative adversarial network (GAN)**.
+It is a kind of generative model with deep neural network, and often applied to
```

Comment:
A GAN is a kind of deep neural network for generative modeling that is often applied to image generation.